operations/system_layout_service/Add_UAN_CAN_IP_Addresses_to_SLS.md

```diff
@@ -73,26 +74,27 @@ This procedure requires administrative privileges.
 
     Example output:
 
-    ```
-    Server:	10.92.100.225
-    Address:	10.92.100.225#53
+    ```console
+    Server:     10.92.100.225
+    Address:    10.92.100.225#53
 
-    Name:	uan01.can
-    Address: 10.103.2.24
+    Name:       uan01.can
+    Address:    10.103.2.24
     ```
 
-    ```
+    ```bash
     ncn-m001# nslookup uan01-can.can
```

Comment:
Do `uan01-can.can` and `uan01-can` exist intentionally? Do we need to have `uan01-can.can` in the document at all?

Reply:
I want to check with Jeff to see if this procedure is still needed.   But I see Jeff is on leave so I left it in for now.